### ISSUE TITLE
React Native Conference sample application not connecting, behaving unexpected and throwing errors

### ISSUE THREAD

**Issue by timantmedia** [ant-media/Ant-Media-Server#5764]
### Short description
The React Native sample application loads fine but is behaving in an unpredictable way. 

For instance, a customer has a reproduce scenario of:

1. Two Android devices connected to a room it is fine
2. both devices disconnect and then reconnect
3. observe that no audio is heard from the Android devices
4. observe audio is heard from the tracks in the web panel

Upon trying to reproduce this issue, connecting to a room using the conference sample application, I experienced it connects briefly, then disconnecting and throwing an error of `ERROR callbackError no_active_streams_in_room `.

Another time, I connected successfully and then added a participant using the desktop browser, the android device is not shown in the room in the browser, but the desktop device is shown in the room on the android device. 

### Environment

* Operating system and version: Android 11
* Java version: 
* Ant Media Server version: 2.6.4
* Browser name and version: Chrome 118

### Steps to reproduce
1. Two Android devices connected to a room it is fine
2. both devices disconnect and then reconnect
3. observe that no audio is heard from the Android devices
4. observe audio is heard from the tracks in the web panel

### Expected behavior
Should always hear audio of the participants in the room and should see all participants regardless of what device they are connecting (mobile app, desktop, laptop)

### Actual behavior
The audio was not heard when reconnecting two Android devices, and the unexpected behavior of not seeing all participants and not connecting after an error.

 

### Logs
__Place logs on [pastebin](http://pastebin.com/) or elsewhere and put links here__

Ask your questions on [Ant Media Github Discussions](https://github.com/orgs/ant-media/discussions)

**Comment by timantmedia** [comment]
I reproduced a few issues here using one emulator and one physical device.

1. open the React Native Android app in the emulator and physical device and connect to multitrack conference
2. Observe audio is working on both.
3. disable audio and re-enable audio on either device does not resume the sound/mic
4. leaving the room and then joining again, the video is frozen in both devices but audio does resume
5. Go to the main page and then click on the multitrack conference and join a room on both devices everything works fine. 


One thing to note is that my video is not in the correct rotation and there are constant messages popping up all the time. 

<img width="515" alt="Screenshot 2023-11-09 at 12 50 50" src="https://github.com/ant-media/Ant-Media-Server/assets/121940452/56da7b5d-3714-4114-88a5-f9775160acc4">

**Comment by SebastianNarvaez11** [comment]
same problem

**Comment by mustafaboleken** [comment]
I fixed the conference sample but there are some issues regarding the reported items.

- These tests are done on multi-track conference applications but React Native only supports old conference architecture.
- The old conference structure also faces some issues with no stream in the room callback and I just fixed it.
- The audio issue that you raised is not reproduced on my side and then I realize that you are using the Native Android SDK not React Native(according to the image that you sent) Android SDK conference sample audio issue is solved in this pull request https://github.com/ant-media/WebRTC-Android-SDK/pull/42
